lib/components/popups/course-search/cart/FineRequirementItem.tsx

```diff
@@ -19,15 +19,15 @@ const FineRequirementListItem: FC<{
   return (
     <div
       className={clsx(
-        props.selected ? 'bg-secondary bg-opacity-25' : 'bg-white',
+        props.selected
+          ? 'bg-secondary bg-opacity-25'
+          : props.itemRequirement.required_credits > 0 &&
+            props.itemRequirement.fulfilled_credits > 0 &&
```

Comment:
is required_credits ever 0?

Reply:
Not sure, but I think this is a good check to have regardless.